scheduler/src/cook/mesos/util.clj

```diff
@@ -338,21 +344,73 @@
 (def ^:const job-states #{"running" "waiting" "completed"})
 (def ^:const instance-states #{"success" "failed"})
 
+
+(defn job-submit-time-between
+  "Returns true if the job-ent's :job/submit-time is between start-ms (inclusive) and
+  end-ms (exclusive)"
+  [job-ent ^long start-ms ^long end-ms]
+  (let [^Date submit-time (:job/submit-time job-ent)
```

Comment:
Can we destructure this from the function arg?

Reply:
Done.